subscribie/blueprints/admin/__init__.py

```diff
@@ -681,14 +681,26 @@ def formatted_status(stripe_external_id):
 @login_required
 def subscribers():
     page = request.args.get("page", 1, type=int)
+    action = request.args.get("action")
 
-    people = (
-        database.session.query(Person)
-        .order_by(desc(Person.created_at))
-        .paginate(page=page, per_page=5)
-    )
+    show_active = action == "show_active"
+
+    def url_pagination(page=None):
+        return url_for("admin.subscribers", page=page, action=action)
+
+    query = database.session.query(Person).order_by(desc(Person.created_at))
```

Comment:
Please
- Restore `per_page=5` because showing two results is too few

- Restore `order_by` created at desc, explicit is better than implicit

Reply:
- restore `order_by` desc created at on pagination query because explicit better than implicit